py3status/modules/file_status.py

```diff
@@ -1,22 +1,45 @@
 # -*- coding: utf-8 -*-
 """
-Display if a file or directory exists.
+Display if a files or directories exists.
 
 Configuration parameters:
     cache_timeout: how often to run the check (default 10)
-    format: format of the output. (default '{icon}')
-    icon_available: icon to display when available (default '●')
-    icon_unavailable: icon to display when unavailable (default '■')
-    path: the path to a file or dir to check if it exists (default None)
+    format: format of the output. (default '\?color=paths [\?if=paths ●|■]')
+    format_path: format of the path output. (default '{basename}')
+    format_path_separator: show separator if more than one (default ' ')
+    path: the path(s) to a file or dir to check if it exists, take a list (default None)
```

Comment:
Too long imo. `path: specify a string or a list of paths to check for (default None)`